### PR TITLE
Fix capitalisation and inconsistencies

### PR DESCRIPTION
**Pull request by BlueGhostGH**
This PR makes sure PureScript and esbuild are properly capitalised
It also addresses certain inconsistencies and mistakes in the README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# Purescript esbuild plugin
+# PureScript esbuild plugin
 
-[Esbuild](https://esbuild.github.io/) integration for [purescript](https://www.purescript.org/)
+[ssbuild](https://esbuild.github.io/) integration for [PureScript](https://www.purescript.org/)
```
Comment (thomashoneyman) [inline README.md:3]: Should this be `esbuild`?

**Comment by thomashoneyman** [comment]
Also, it looks like the `package-lock.json` file was committed, so there are two different lockfiles in the repo now -- is that desired?

File: README.md (hunk 1 above)
Comment (BlueGhostGH) [reply README.md:3]: Yes, my bad 😓

**Comment by BlueGhostGH** [comment]
> Also, it looks like the `package-lock.json` file was committed, so there are two different lockfiles in the repo now -- is that desired?

Nope, it's not. I have used GitHub Codespaces to quickly edit the repo since it was supposed to be just a few changes but didn't notice Codespaces ran an npm install for me and generated an npm lock file. It is my fault for not checking I commit only the files I want to commit.

This is a bit embarrassing but thanks for pointing it out

**Pull request merged**